fasttext.py

```diff
@@ -53,6 +53,24 @@ def apply_transform(self, transform):
         transmat = np.loadtxt(transform) if isinstance(transform, str) else transform
         self.embed = np.matmul(self.embed, transmat)
 
+    def export(self, outpath):
+        """
+        Transforming a large matrix of WordVectors is expensive. 
+        This method lets you write the transformed matrix back to a file for future use
+        :param The path to the output file to be written 
+        """
+        fout = open(outpath, "w")
+
+        # Header takes the guesswork out of loading by recording how many lines, vector dims
+        fout.write(str(self.n_words) + " " + str(self.n_dim) + "\n")
+        for k in self.word2id.keys():
+            vector_components = ["%.6f" % number for number in self[k]]
```

Comment:
Looks right, based on a few seconds of glancing at the original files.  Didn't see any values that needed more precision than this, and in general, DNN weights and representations tend not to be demanding in terms of precision (I'm sure you've seen the shift to 16-bit floats and lower in hardware from Intel/NVIDIA/Google etc.).